config/set/laravel.php

```diff
@@ -0,0 +1,377 @@
+        'groups' => [
+            ['deprecated', 'link', 'see', 'since'],
+            ['author', 'copyright', 'license'],
+            ['category', 'package', 'subpackage'],
+            ['property', 'property-read', 'property-write'],
+            ['param', 'return'],
+        ],
+    ]);
+    $ecsConfig->rule(PhpdocSingleLineVarSpacingFixer::class);
+    $ecsConfig->ruleWithConfiguration(PhpdocTagTypeFixer::class, [
+        'tags' => ['inheritdoc' => 'inline'],
+    ]);
+    $ecsConfig->rules([
+        PhpdocTrimFixer::class,
+        PhpdocTypesFixer::class,
+        PhpdocVarWithoutNameFixer::class,
+    ]);
+    $ecsConfig->ruleWithConfiguration(ReturnTypeDeclarationFixer::class, [
+        'space_before' => 'none'
+    ]);
+    $ecsConfig->rules([
+        SelfStaticAccessorFixer::class,
+        ShortScalarCastFixer::class,
+        SingleBlankLineAtEofFixer::class,
+        SingleBlankLineBeforeNamespaceFixer::class,
+    ]);
+    $ecsConfig->ruleWithConfiguration(SingleClassElementPerStatementFixer::class, [
+        'elements' => ['const', 'property'],
+    ]);
+    $ecsConfig->rules([
+        SingleImportPerStatementFixer::class,
+        SingleLineAfterImportsFixer::class,
+    ]);
+    $ecsConfig->ruleWithConfiguration(SingleLineCommentStyleFixer::class, [
+        'comment_types' => ['hash'],
+    ]);
+    $ecsConfig->rules([
+        SingleQuoteFixer::class,
+        SingleSpaceAroundConstructFixer::class,
+        SpaceAfterSemicolonFixer::class,
+        StandardizeNotEqualsFixer::class,
+        SwitchCaseSemicolonToColonFixer::class,
+        SwitchCaseSpaceFixer::class,
+        TernaryOperatorSpacesFixer::class,
+    ]);
+    $ecsConfig->ruleWithConfiguration(TrailingCommaInMultilineFixer::class, [
+        'elements' => ['arrays'],
+    ]);
+    $ecsConfig->rules([
+        TrimArraySpacesFixer::class,
+        TypesSpacesFixer::class,
+        UnaryOperatorSpacesFixer::class,
+    ]);
+    $ecsConfig->ruleWithConfiguration(VisibilityRequiredFixer::class, [
+        'elements' => ['method', 'property'],
+    ]);
+    $ecsConfig->rule(WhitespaceAfterCommaInArrayFixer::class);
+
+    // Laravel custom fixer
+    $ecsConfig->rule(LaravelPhpdocAlignmentFixer::class);
+
+    // any rules marked as 'false' are skipped
+    $ecsConfig->skip([
+        PhpdocSummaryFixer::class,
+        PhpdocToCommentFixer::class,
+        PsrAutoloadingFixer::class,
+        SelfAccessorFixer::class,
+        SimplifiedNullReturnFixer::class,
+        StatementIndentationFixer::class,
+        // App\Factories\ConfigurationFactory::$notName
+        '_ide_helper*.php',
+        '.phpstorm.meta.php',
+        '*.blade.php',
+        // App\Factories\ConfigurationFactory::$exclude
+        __DIR__ . 'bootstrap/cache',
+        __DIR__ . 'build',
+        __DIR__ . 'node_modules',
+        __DIR__ . 'storage',
+    ]);
+
+    // @todo deal with default path configuration & any other default configs for the set?
```

Comment:
@TomasVotruba it seems like the other setlists don't use any `paths()` conventions or similar, so should I just remove the comment here and call it done?